doc/manual/noteworthy-differences.rst

```diff
@@ -367,3 +367,16 @@ Noteworthy differences from C/C++
   in order to have dynamic dispatch.
   On the other hand, in Julia every method is "virtual" (although it's more general than that
   since methods are dispatched on every argument type, not only ``this``, using the most-specific-declaration rule).
+
+Noteworthy differences from Common Lisp
+----------------------------------------
+
+- Julia uses 1-based indexing for arrays.
+- Functions and variables share the same namespace ("Lisp-1")
+- There is a :obj:`Pair` type, but it is not meant to be used as a ``cons``. Various iterable collections can be used interchangeably in most parts of the language (eg splat, tuples, etc).
+- Julia does not encourage continuous manipulation of the image and hotswapping, reloading the workspace is frequently necessary (see also `#265 <https://github.com/JuliaLang/julia/issues/265>`_).
```

Comment:
I'd use a ; after "hotswapping", to avoid a [run-on sentence](https://en.wikipedia.org/wiki/Run-on_sentence).